boards/posix/native_posix/Kconfig

```diff
@@ -26,4 +26,26 @@ config HAS_SDL
 	help
 	  This option specifies that the target board has SDL support
 
-endif
+if HAS_SDL
+
+config SDL_THREAD_INTERVAL
+	int "SDL thread sleep interval"
+	default 10
+	help
+	  Sleep interval time of SDL thread to handle events in microseconds.
+
+config SDL_THREAD_PRIORITY
+	int "SDL thread priority"
+	default 0
+	help
+	  Priority of SDL thread to handle events.
+
+config SDL_THREAD_STACK_SIZE
+	int "SDL thread stack size"
+	default 2048
```

Comment:
There is no need to define a stack size for this. You can just use ARCH_POSIX_RECOMMENDED_STACK_SIZE See https://github.com/zephyrproject-rtos/zephyr/blob/main/arch/posix/Kconfig#L12